### PR TITLE
Removed call to analytics.js and added GTM tracking code on the Academy landing page

### PR DESCRIPTION
**Pull request by thomasmalbaux**

### CODE REVIEW

File: src/html/pages/academy/index.html [hunk 1]
```diff
@@ -57,10 +57,17 @@
   <script src="https://use.typekit.net/muu2ohi.js"></script>
   <script>try{Typekit.load({ async: false });}catch(e){}</script>
 
-  <script><!--(bake /includes/analytics.js)--></script>
```
Comment (vsaarinen) [inline src/html/pages/academy/index.html]: analytics.js includes all of our analytics. Do we want to remove all of them?

If not, adding an if-clause to the bake command whether to not include analytics.js is probably the best way to go.

**Comment by thomasmalbaux** [comment]
Correct. Unfortunately I don't feel comfortable with adding those bake clauses so my thinking was that it wouldn't be critical to "lose" retargeting info on the Academy landing page (only) until soft launch.
I should have mentioned this in the description.

If anyone is OK to build this clause that would be perfect but if not, I suggest to "lose" it for a couple days.

What do you think?

**Comment by vsaarinen** [comment]
Removing analytics.js will also cause an error when you click on the subscribe button because we track that with Google Analytics. Is it ok to have both Google Analytics and GTM on the page?

**Comment by thomasmalbaux** [comment]
OK let's forget this. I'll wait for soft launch to finalize this, definitely not critical anyway.

**Pull request closed without merging**